### PR TITLE
Update 001-electricity-meter-monitoring.zh-hans.md

### PR DESCRIPTION
**Pull request by fenghuazzm**
1. 更新超链接 link；
2. 补充了建模语句。

### CODE REVIEW

File: cases/001-electricity-meter-monitoring.zh-hans.md [hunk 1]
```diff
@@ -42,11 +42,24 @@ CREATE STABLE meters (ts timestamp, current float, voltage int, phase float) TAG
 CREATE TABLE d1001 USING meters TAGS ("Beijing.Chaoyang", 2);
 ```
 
+* 更改标签值。有时因为需求、设备变更需要更改子表标签值：
+
+```
+ALTER TABLE d1001 SET TAG  groupid = 1;
```
Comment (ericyangpan) [inline cases/001-electricity-meter-monitoring.zh-hans.md]: additional space

File: cases/001-electricity-meter-monitoring.zh-hans.md [hunk 2]
```diff
@@ -42,11 +42,24 @@ CREATE STABLE meters (ts timestamp, current float, voltage int, phase float) TAG
...
+```
+
+* 查看超级表、子表结构。执行后可以看到子表继承了其超级表的结构：
+
+```
+describe meters;
```
Comment (ericyangpan) [inline cases/001-electricity-meter-monitoring.zh-hans.md]: All the SQL statements should be capitalized.

Comment (ericyangpan) [reply cases/001-electricity-meter-monitoring.zh-hans.md]: All the SQL keywords should be capitalized.

File: cases/001-electricity-meter-monitoring.zh-hans.md [hunk 3]
```diff
@@ -100,7 +113,7 @@ SELECT * FROM meters;
 SELECT d1001.* FROM d1001, d1003 WHERE d1001.ts=d1003.ts;
 ```
 
-* 函数查询，count(*) 函数只返回一列，first 、last 、last_row 函数返回全部列：
+* 函数查询，[戳我查看 TDengine 支持的函数](https://docs.taosdata.com/taos-sql/function)。count(*) 函数只返回一列，first 、last 、last_row 函数返回全部列：
```
Comment (ericyangpan) [inline cases/001-electricity-meter-monitoring.zh-hans.md]: All the SQL keywords should be capitalized.

File: cases/001-electricity-meter-monitoring.zh-hans.md [hunk 4]
```diff
@@ -126,5 +139,5 @@ SELECT DISTINCT current , voltage , phase  FROM d1001;
 # 数据展示
 TDengine 服务启动后会自动创建一个监测数据库 log ，包含对服务器的一系列监控数据，通过 Grafana 和 TDengine 数据源插件，构成了数据库监控、可视化解决方案 TDinsight 。
 
-详细介绍与部署操作请移步：[TDinsight - 基于Grafana的TDengine零依赖监控解决方案](https://www.taosdata.com/docs/cn/v2.0/tools/insight)
+详细介绍与部署操作请移步：[TDinsight - 基于Grafana的TDengine零依赖监控解决方案](https://docs.taosdata.com/reference/tdinsight/)
```
Comment (ericyangpan) [inline cases/001-electricity-meter-monitoring.zh-hans.md]: Space between Chinese char and English char is needed.

**Review by ericyangpan — CHANGES_REQUESTED** [review]
Need more changes

**Comment by fenghuazzm** [comment]
Done

File: cases/001-electricity-meter-monitoring.zh-hans.md [hunk 5]
```diff
@@ -100,19 +113,19 @@ SELECT * FROM meters;
 SELECT d1001.* FROM d1001, d1003 WHERE d1001.ts=d1003.ts;
 ```
 
-* 函数查询，count(*) 函数只返回一列，first 、last 、last_row 函数返回全部列：
+* 函数查询，[戳我查看 TDengine 支持的函数](https://docs.taosdata.com/taos-sql/function)。COUNT(*) 函数只返回一列，FIRST 、LAST 、LAST_ROW 函数返回全部列：
```
Comment (ericyangpan) [inline cases/001-electricity-meter-monitoring.zh-hans.md]: additional space after FIRST & LAST.

**Review by ericyangpan — CHANGES_REQUESTED** [review]
Need more changes

**Review by ericyangpan — APPROVED** [review]

**Review by ericyangpan — APPROVED** [review]

**Pull request merged**